### PR TITLE
Feature 740 theme service improvements

### PR DESCRIPTION
**Pull request by tlainez**
# PR Details

In order to complete the implementation of the theme service, there were some pending tasks, specially for the "dark" theme.
All the changes affect mainly to the scss associated to each component.

## Description

- Replace color variables from SASS to CSS
- Define additional required CSS variables
- Review the correct implementation in both current themes (default & dark) as well as if no theme is specified
- Reorder the styles defined in the angular.json to grant that the custom styles prevail over the bootstrap defaults

## Related Issue

https://github.com/systelab/systelab-components/issues/740

## Motivation and Context

Complete the theme service implementation and define the "dark" theme color palette.

## How Has This Been Tested

The changes have been tested in the showcase 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] I have read the **CONTRIBUTING** document
- [X] My code follows the code style of this project
- [ ] My change requires a change to the documentation 
- [ ] I have updated the documentation accordingly (README.md for each UI component)
- [ ] I have added tests to cover my changes (at least 1 spec for each UI component with the same coverage as the master branch)
- [ ] All new and existing tests passed
- [ ] A new branch needs to be created from master to evolve previous versions
- [X] Increase version in package.json following [Semantic Versioning](https://semver.org/)
- [ ] All UI components must be added into the showcase (at least 1 component with the default settings)
- [X] Add the issue into the right [project](https://github.com/systelab/systelab-components/projects) with the proper status (In progress)

### CODE REVIEW

**Review by oGalceran1 — APPROVED** [review]
Thanks

File: projects/systelab-components/sass/modern/_button.scss [hunk 1]
```diff
@@ -22,28 +28,32 @@
   border-radius: 20px;
 }
 
-systelab-toggle-button {
-  .btn { //Toogle buttons
-    &.btn-outline-primary {
-      color: $primary;
-      background-color: transparent;
-      border-color: $primary;
+.btn {
```
Comment (OscarDeDios) [inline projects/systelab-components/sass/modern/_button.scss:31]: why did you remove systelab-toggle-button selector?

now it's a generic stile for btn-outline-primary?

Comment (tlainez) [reply projects/systelab-components/sass/modern/_button.scss:31]: Yes, these styles should apply also every btn-outline-primary, so I have made them generic

**Review by OscarDeDios — APPROVED** [review]
